medusa/providers/torrent/json/danishbits.py

```diff
@@ -0,0 +1,148 @@
+# coding=utf-8
+
+"""Provider code for Danishbits."""
+
+from __future__ import unicode_literals
+
+import logging
+import traceback
+
+from medusa import tv
+from medusa.helper.common import convert_size
+from medusa.logger.adapters.style import BraceAdapter
+from medusa.providers.torrent.torrent_provider import TorrentProvider
+
+from requests.compat import urljoin
+
+log = BraceAdapter(logging.getLogger(__name__))
+log.logger.addHandler(logging.NullHandler())
+
+
+class DanishbitsProvider(TorrentProvider):
+    """Danishbits Torrent provider."""
+
+    def __init__(self):
+        """Initialize the class."""
+        super(DanishbitsProvider, self).__init__('Danishbits')
+
+        # Credentials
+        self.username = None
+        self.passkey = None
+
+        # URLs
+        self.url = 'https://danishbits.org'
+        self.urls = {
+            'search': urljoin(self.url, 'couchpotato.php'),
+        }
+
+        # Proper Strings
```

Comment:
Means it uses the ones from generic. Which you reminded me to check if they work on the API.